### PR TITLE
ru: day 2, afternoon; more control loops translation

### PR DESCRIPTION
**Pull request by Makarevich**

### CODE REVIEW

**Review by dyeroshenko — APPROVED** [review]
LGTM

**Comment by mgeisler** [comment]
Hi @Makarevich, I caused the merge conflicts here :grimacing: Please run

```shell
cargo install mdbook-i18n-normalize
```

and see the [usage instructions](https://github.com/google/mdbook-i18n-helpers/blob/main/USAGE.md#normalizing-existing-po-files) for how to use `mdbook-i18n-normlize` to update your local `ru.po` file to the new format. Commit this and your PR should stop having conflicts.

**Comment by mgeisler** [comment]
Hi all, please merge this after resolving the conflicts.

**Comment by mgeisler** [comment]
@Makarevich, please merge this now that it has been approved.

File: po/ru.po [hunk 1]
```diff
@@ -4356,6 +4356,19 @@ msgid ""
 "}\n"
 "```"
 msgstr ""
+"```rust,editable\\n\n"
```
Comment (mgeisler) [inline po/ru.po:4359]: I believe `\\n\n` should be just `\n` here and below.

File: po/ru.po [hunk 2]
```diff
@@ -4356,6 +4356,19 @@ msgid ""
...
+"fn main() {\\n\n"
+"    let mut x = 10;\\n\n"
+"    while x != 1 {\\n\n"
+"        x = if x % 2 == 0 {\\n\n"
+"            x / 2\\n\n"
+"        } else {\\n\n"
+"            3 * x + 1\\n\n"
+"        };\\n\n"
+"    }\\n\n"
+"    println!(\\\"Конечное значение x: {x}\\\");\\n\n"
```
Comment (mgeisler) [inline po/ru.po:4369]: You've managed to escape the backslash: `\\\"` should be just `\"`.

File: po/ru.po (hunk 1 above)
Comment (Makarevich) [reply po/ru.po:4359]: Duh! I had to manually fix the merge and copied too many backslashes. :) Let me fix that.

Comment (mgeisler) [reply po/ru.po]: Thanks!

**Pull request merged**